renovate.json

```diff
@@ -20,6 +20,14 @@
       "fileMatch": ["^.github/workflows/continuous-integration.yml$"],
       "matchStrings": ["(?<depName>cypress/included):(?<currentValue>[0-9.]+)\n"],
       "datasourceTemplate": "docker"
+    },
+    {
+      "fileMatch": ["^renovate.json$"],
+      "matchStrings": [
+        "\"node\": \"= (?<latestVersion>.+?)"
+      ],
+      "depNameTemplate": "node",
+      "datasourceTemplate":"git-tags"
```

Comment:
Git Tags klingt theoretisch gut, keine Ahnung ob Renovate weiss, wo das Git Repo von node liegt, vielleicht wäre auch `github-tags` passender, und `depNameTemplate` auf `nodejs/node` ändern. Das Ziel wäre dann, dass Renovate diese Liste sieht: https://github.com/nodejs/node/tags

Die Tags haben aber noch ein `v` vorne dran, das wir noch entfernen müssen, bevor Renovate die Version in die Config einfügen kann. Dafür ist wohl `extractVersionTemplate` gedacht, aber das funktioniert im Moment nicht https://github.com/renovatebot/renovate/pull/10325

Gibt es irgendwo sonst noch eine Liste von Node Versionen, ohne v vorne dran?

Reply:
Ah, ein Workaround ist in https://github.com/renovatebot/renovate/issues/8842 beschrieben. Wir könnten anscheinend zusätzlich folgende `packageRule` hinzufügen:
```
  "packageRules": [
    {
      "matchManagers": ["regex"],
      "extractVersion": "^v?(?<version>.+)$"
    }
  ],
```

Reply:
You can make more specific rules (like, per package name or somehow else), my example just worked quite fine for me :)

Reply:
@okainov thanks for the input! It seems that almost at the time of you writing that, the PR renovatebot/renovate#10325 was merged. So in theory, extractVersionTemplate should now work.
I say "in theory" because the PR looks to me like it only changes the validation, but I haven't yet found any code in renovate that actually implements this...

Reply:
Habe das auto update der node version von renovate entfernt.
Können wir später mal machen